### DISCUSSION TITLE
post banner 设置后不起作用

### DISCUSSION THREAD

**Discussion by ssy666** [ppoffice/hexo-theme-icarus#909] [Q&A]
今天装了您的主题，在 post 头文件中设置 thumbnail 和 banner，thumbnail 可以正常显示，但 banner 始终显示不了。

我设置的方式是：

```
---
title: Hello World
banner: https://pic2.zhimg.com/80/v2-a0f6f77bffc0b3d4d53351b0b86130e3_720w.jpg
thumbnail: https://pic2.zhimg.com/80/v2-a0f6f77bffc0b3d4d53351b0b86130e3_720w.jpg
tags: 新的
---

```

查阅了已关闭的 issue，发现也有人提过这个问题。

**Comment by ppoffice** [comment]
@ssy666 https://ppoffice.github.io/hexo-theme-icarus/uncategorized/%E5%8D%87%E7%BA%A7%E6%8C%87%E5%8D%97/
